### DISCUSSION TITLE
how to clear  Text  Create By new PIXI.Text?

### DISCUSSION THREAD

**Discussion by Caesar454905970** [pixijs/pixijs#8882] [Q&A]
const richText = new PIXI.Text('Rich text with a lot of options and across multiple lines', style);
richText.x = 50;
richText.y = 220;

app.stage.addChild(richText);

I need to constantly change the content and position of the text

**Comment by SuperSodaSea** [comment]
You just need to change `richText.x`, `richText.y` and `richText.text`, and Pixi will do the rest for you.

Here is an example: https://www.pixiplayground.com/#/edit/cWj_mPJJAAyelX26R9cAD

```js
const app = new PIXI.Application({
    width: window.innerWidth,
    height: window.innerHeight,
    backgroundColor: 0x2c3e50,
});
document.body.appendChild(app.view);

const textStyle = new PIXI.TextStyle({
    fontFamily: 'Arial',
    fontSize: 36,
    fill: '#ffffff',
});

const text = new PIXI.Text('', textStyle);
text.anchor.set(0.5);
app.stage.addChild(text);

let v = 0;
app.ticker.add((delta) => {
    v += delta;
    text.x = 200 + 100 * Math.cos(0.025 * v);
    text.y = 200 + 100 * Math.sin(0.025 * v);
    text.text = 'Time: ' + (0.001 * performance.now()).toFixed(3);
});
```

**Reply by Caesar454905970** [reply]
thank you !

**Reply by bigtimebuddy** [reply]
Keep in mind, constantly changing the content of text is expensive. For every change, it has to upload a new texture to the GPU which can slow down your whole app. Consider using BitmapText and BitmapFont instead, especially if you have a small character set.

**Reply by Caesar454905970** [reply]
thank you for your notice
